invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitiesListModal/RecordCommunitiesSearchItem.js

```diff
@@ -1,24 +1,171 @@
-import React from "react";
+import React, { Component } from "react";
 import { CommunityCompactItem } from "@js/invenio_communities/community";
-import { Button } from "semantic-ui-react";
+import { Button, Modal, Message, Icon, Checkbox, Grid } from "semantic-ui-react";
 import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { Trans } from "react-i18next";
+import { http, GeneralErrorMessage } from "react-invenio-forms";
 import PropTypes from "prop-types";
+import { withState } from "react-searchkit";
 
-export const RecordCommunitiesSearchItem = ({ result }) => {
-  const action = (
-    <Button
-      disabled
-      size="tiny"
-      negative
-      labelPosition="left"
-      icon="trash"
-      floated="right"
-      content={i18next.t("Remove")}
-    />
-  );
-  return <CommunityCompactItem actions={action} result={result} />;
-};
+class RecordCommunitiesSearchItemComponent extends Component {
+  initialErrorState = undefined;
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalOpen: false,
+      loading: false,
+      error: this.initialErrorState,
+      isButtonDisabled: true,
+      isChecked1: false,
+      isChecked2: false,
+    };
+  }
+
+  handleCheckbox1Change = () => {
+    this.setState({ isChecked1: !this.state.isChecked1 });
+    this.setState({
+      isButtonDisabled: !(this.state.isChecked2 && !this.state.isChecked1),
+    });
```

Comment:
Minor: It can be written in one line:
```suggestion
    this.setState({ isChecked1: !this.state.isChecked1, isButtonDisabled: !(this.state.isChecked2 && !this.state.isChecked1) });
```

Same for below.